packages/design-system/src/components/web-components/ds-autocomplete/ds-autocomplete.tsx

```diff
@@ -73,7 +82,20 @@ const Wrapper = ({
       items={parseJsonAttr(items)}
       loading={parseBooleanAttr(loading)}
     >
-      <TextField label={label} hint={hint} name="autocomplete" value={value} />
+      <TextField
+        disabled={parseBooleanAttr(otherProps.disabled)}
+        errorMessage={otherProps.errorMessage}
+        errorPlacement={
+          isPossibleValue(otherProps.errorPlacement, ['top', 'bottom'])
+            ? otherProps.errorPlacement
+            : null
```

Comment:
Looks like this needs to change to `undefined` in order to support the default error placement based on each theme's [configuration](
https://github.com/CMSgov/design-system/blob/jryan/wnmgds-2992-ds-autocomplete-pass-through-props-to-text-field/packages/design-system/src/components/InlineError/useInlineError.tsx#L50).

When passing along `null`, that value is used instead of calling that `config()` function for the default.

Reply:
Good catch. Thank you for flagging.